### DISCUSSION TITLE
请问我的管理员设置有错误吗

### DISCUSSION THREAD

**Discussion by xzzt1267** [FloatTech/ZeroBot-Plugin#420] [Q&A] [answered]
我在代码里写入了自己的QQ号，但是还是设置不了搜图的api

<img width="367" alt="代码" src="https://user-images.githubusercontent.com/96318711/190844997-135afa92-d588-4bc5-b514-f8a7adfc0083.png">
<img width="559" alt="设置" src="https://user-images.githubusercontent.com/96318711/190845002-5b4da391-b7c7-4dbd-9af3-70dbd9c84352.png">

**Comment by guohuiyuan** [comment, marked as answer]
不要带[]
